rosserial_arduino/setup.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
Adding Python modules to `rosserial_arduino` requires a bunch of changes to the `CMakeLists.txt` and `package.xml` files. Please ensure those are included here.

Reply:
I think these should now be up to date. Please let me know if I missed anything.